locale/fr.json

```diff
@@ -236,24 +236,24 @@
 	"TZ": "Tanzanie",
 	"UA": "Ukraine",
 	"UG": "Ouganda",
-	"UM": "Iles mineures éloignées des Etats-Unis",
-	"US": "Etats-Unis d'Amérique",
+	"UM": "Iles mineures éloignées des États-Unis",
+	"US": "États-Unis d'Amérique",
 	"UY": "Uruguay",
 	"UZ": "Ouzbékistan",
 	"VA": "Vatican",
 	"VC": "St. Vincent et Grenadines",
 	"VE": "Venezuela",
 	"VG": "Vierges, Iles britanniques (Tortola)",
-	"VI": "Iles Vierges (Etats-Unis)",
+	"VI": "Îles Vierges (États-Unis)",
 	"VN": "Viêt-Nam",
 	"VU": "Vanuatu",
 	"WF": "Wallis et Futuna",
-	"WS": "Samoa, occidental",
+	"WS": "Samoa",
 	"XK": "Kosovo",
 	"YE": "Yemen",
 	"YT": "Mayotte",
 	"ZA": "Afrique du Sud",
 	"ZM": "Zambie",
 	"ZW": "Zimbabwe",
-	"ZZ": "Internacional"
-}
\ No newline at end of file
+	"ZZ": "International"
```

Comment:
Are you sure it's not "Internationale"?

Reply:
Yep. 
I’m a french speaker. Here’s an example in context : https://www.voyagevoyage.ca/conseils/application-appeler-international-pas-cher-1.2487299

But it’s definitely not "Internacionale". All the other corrections come directly from Wikipedia Fr or my own knowlegde (backed by Wikipedia)

Reply:
@FredChauviere Ok, merging.

Reply:
But for the rest, awesome job! Thank you!